UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -70,32 +66,83 @@ namespace UnitsNet
         [DataMember(Name = "Unit", Order = 2)]
         private readonly AccelerationUnit? _unit;
 
-        static Acceleration()
+        /// <summary>
+        ///     Provides detailed information about the <see cref="Acceleration"/> quantity, including its name, base unit, unit mappings, base dimensions, and conversion functions.
+        /// </summary>
+        public sealed class AccelerationInfo: QuantityInfo<Acceleration, AccelerationUnit>
         {
```

Comment:
I'm a little on the fence on the usefulness of having a concrete class for each quantity's info, and we have 100+ quantities that will add to the binary size.

If it's about syntax, a type alias would achieve much the same.
I don't expect to pass `AccelerationInfo` around or something consumers of UnitsNet should think about.
For extensibility, it doesn't seem to offer much.